File: examples-sdk/swift/101c-vilapp-multi-pipeline-benchmark/main.swift
// 101c-vilapp-multi-pipeline-benchmark — Swift SDK equivalent
// Compile: vil compile --from swift --input 101c-vilapp-multi-pipeline-benchmark/main.swift --release

let server = VilServer(name: "multi-pipeline-bench", port: 3090)
let pipeline = ServiceProcess(name: "pipeline")
server.service(pipeline)
server.compile()

File: examples-sdk/csharp/101c-vilapp-multi-pipeline-benchmark/Main.cs
// 101c-vilapp-multi-pipeline-benchmark — C# SDK equivalent
// Compile: vil compile --from csharp --input 101c-vilapp-multi-pipeline-benchmark/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("multi-pipeline-bench", 3090);
var pipeline = new ServiceProcess("pipeline");
server.Service(pipeline);
server.Compile();
